README.md

```diff
@@ -116,7 +116,7 @@ FROM
     TotalRecords, OutlierCount;
 -->
 
-As of 2024-02-28, around **6.9%** of the total sales have some sort of outlier classification. Within that 6.9%, the proportion of each outlier type is:
+As of 2024-03-15, around **6.9%** of the total sales have some sort of outlier classification. Within that 6.9%, the proportion of each outlier type is:
```

Comment:
**[Question, non-blocking]** Is the 6.9% number still correct given the changes to the percentages in the table below?

Reply:
Yep!